Babylon/commands/api/workspaces/delete.py

```diff
@@ -1,54 +1,57 @@
 from logging import getLogger
 from typing import Any
-from click import Context, command, pass_context
-from click import argument
+from click import command
 from click import option
+
+from Babylon.commands.api.workspaces.service.api import WorkspaceService
 from Babylon.utils.interactive import confirm_deletion
-from Babylon.utils.decorators import inject_context_with_resource, wrapcontext
+from Babylon.utils.decorators import (
+    wrapcontext,
+    retrieve_state,
+)
 from Babylon.utils.decorators import timing_decorator
 from Babylon.utils.environment import Environment
 from Babylon.utils.response import CommandResponse
 from Babylon.utils.credentials import pass_azure_token
-from Babylon.utils.request import oauth_request
-from Babylon.utils.typing import QueryType
 
 logger = getLogger("Babylon")
 env = Environment()
 
 
 @command()
 @wrapcontext()
-@pass_context
 @timing_decorator
+@retrieve_state
 @pass_azure_token("csm_api")
-@option("-D", "force_validation", is_flag=True, help="Force Delete")
-@argument("id", type=QueryType(), required=False)
-@inject_context_with_resource({"api": ['url', 'organization_id']})
+@option("-D", "force_validation", is_flag=True, default=True, help="Force Delete")
```

Comment:
typo: missing dashes (-) before the long option (`--force_validation`)
also it is easier to memorize if short and long command line arg have some letter in common, perhaps at some point we should have a global look at the commands and rethink if `D` and `force_validation` are the best options

Reply:
❌ ignore the above 😃 I see it was indeed as the variable name